src/main/java/wolox/training/exceptions/UserMismatchException.java

```diff
@@ -0,0 +1,9 @@
+package wolox.training.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.NOT_FOUND, reason = "User id does not match with given id")
```

Comment:
I think the code could be a bad request instead (400 / `HttpStatus.BAD_REQUEST`)